index.js

```diff
@@ -31,40 +31,42 @@ module.exports = function (opts) {
       return progress(state)
     },
     request: function (id, follows) {
-      if(opts.isFeed && !opts.isFeed(id)) return
+      if (opts.isFeed && !opts.isFeed(id)) return
       self.state = events.follow(self.state, {id: id, value: follows !== false, ts: timestamp()})
       self.update()
     },
     pause: function (id, paused) {
-      self.state = events.pause(self.state, {id: id, paused: paused !== false})
+      self.state = events.pause(self.state, {id, paused: paused !== false})
       self.update()
     },
     block: function (id, target, value) {
-      self.state = events.block(self.state, {id: id, target: target, value: value !== false, ts: timestamp()})
+      self.state = events.block(self.state, {id, target, value: value !== false, ts: timestamp()})
       self.update()
     },
-    createStream: function (remote_id, version, client) {
-      if(self.streams[remote_id])
-        self.streams[remote_id].end(new Error('reconnected to peer'))
-      if(self.logging) console.log('EBT:conn', remote_id)
-      var stream = self.streams[remote_id] = new Stream(this, remote_id, version, client, opts.isMsg, function (peerState) {
-        opts.setClock(remote_id, peerState.clock)
-      })
+    createStream: function (remoteId, version, client) {
+      if (self.streams[remoteId])
+        self.streams[remoteId].end(new Error('reconnected to peer'))
+      if (self.logging) console.log('EBT:conn', remoteId)
+      function onClose(peerState) {
+        opts.setClock(remoteId, peerState.clock)
+      }
+      var stream = new Stream(this, remoteId, version, client, opts.isMsg, onClose)
+      self.streams[remoteId] = stream
 
-      opts.getClock(remote_id, function (err, clock) {
+      opts.getClock(remoteId, (err, clock) => {
```

Comment:
This may change the version of node supported?

Reply:
Yeah, but arrow functions are supported since node 6 (released 2016, LTS ended 2019 ;-)). And latest stable is 16. Our automated tests run only 12 and forward.